index.html

```diff
@@ -457,12 +271,13 @@ <h2 class="about_title">About</h2>
         <div class="dot_image_with_teal"></div>
       </div>
     </section>
-    <section class="contact_section" id="Contact">
+    <section class="contact_section can_dissapear" id="Contact">
```

Comment:
- [ ] This bug was introduced on your desktop view at some point, please kindly fix it:

![image](https://user-images.githubusercontent.com/57421823/151078012-3fbabc82-0639-4ac0-9f67-80b92db6d3e8.png)

Reply:
@Meltrust Thank you for catching this, I'm not able to reproduce this issue, the difference might be based on the browser. I would appreciate it if you could specify the browsers you were using 🐱‍👤.

Reply:
You are very right!  In Chrome it works, but not on Firefox.  I had to remove the width on the element in order for it to work:

![image](https://user-images.githubusercontent.com/57421823/151152185-9eadfcdb-33d3-436f-96c9-b46d8a243137.png)

Thanks for pointing this out.

Reply:
@Meltrust Thank you once again for catching this, I'll implement the change 🐱‍🏍.